test/features/can_revert_record_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+Dir[Rails.root.join('test/**/*.rb')].each { |f| require f }
+
+class CanRevertRecordTest < Capybara::Rails::TestCase
+  include Helpers::UserHelpers
+
+  describe 'Reverting Collections' do
+    before do
+      OmniAuth.config.test_mode = true
+      mock_login(role: 'admin')
+
+      stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.echo10?concept_id=G309210-GHRC")
+        .with(
+          headers: { 'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Accept' => '*/*', 'User-Agent' => 'Ruby' }
+        )
+        .to_return(status: 200, body: '<?xml version="1.0" encoding="UTF-8"?><results><hits>0</hits><took>32</took></results>', headers: {})
+    end
+
+    it 'can revert a record, mimics workflow in issue CMRARC-484' do
```

Comment:
If you are going to reference the issue, I think it is worth commenting that this test would not actually catch the error that needed to be fixed.